product_security/i18n/ja.po

```diff
@@ -0,0 +1,37 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* product_security
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 16.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-10-13 05:58+0000\n"
+"PO-Revision-Date: 2023-10-13 05:58+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: product_security
+#: model:ir.model,name:product_security.model_product_template
+msgid "Product"
+msgstr "プロダクト"
+
+#. module: product_security
+#: model:res.groups,name:product_security.group_product_manager
+msgid "Product Manager"
+msgstr "プロダクト管理者"
+
+#. module: product_security
+#. odoo-python
+#: code:addons/product_security/models/product_template.py:0
+#, python-format
+msgid ""
+"You are not allowed to update product category. Please contact administrator"
+" as necessary."
+msgstr ""
+"プロダクトカテゴリーを更新することは許可されていません。"
```

Comment:
```suggestion
"プロダクトカテゴリを更新することは許可されていません。"
```
Just in case, can we keep the translation the same as the standard? 
Sorry for the detail.